mpf/config_spec.yaml

```diff
@@ -582,6 +582,7 @@ hardware_sound_player:
 high_score:
     __valid_in__: mode
     award_slide_display_time: single|ms|4s
+    high_score_reverse_sort: single|bool|False
```

Comment:
Since this config setting is part of `high_score`, is it necessary to prefix the setting name with it? Why not just `reverse_sort`?

Reply:
Good point. I will correct that as well in my next push.